pkg/config/loader_test.go

```diff
@@ -48,7 +60,7 @@ type AwsConfig struct {
 	MaxBackoff int       `hcl:"max_backoff,optional" default:"30"`
 }
 
-func TestParser_LoadConfigFromSource(t *testing.T) {
+func TestLoader_LoadConfigFromSource(t *testing.T) {
```

Comment:
I think these tests should be moved to `config.parser.go` when the code that does the parsing is, but I'll keep that to a future PR